### DISCUSSION TITLE
Alive caption to Android Motorola g play 2023

### DISCUSSION THREAD

**Discussion by Wildcrazy4u** [community/community#135987] [Accessibility]
### Select Topic Area

Question

### GitHub Feature Area

Mobile

### Body

I am deaf so I'd like anything vice it'll pop up closed captions.

**Comment by queenofcorgis** [comment]
Hi @Wildcrazy4u ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Android like their [Support community](https://support.google.com/android/community?hl=en). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions and this category focus primarily on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Wildcrazy4u** [comment]
Hey thank you, I am learning, umm how do I go from there?  by dail it on phone number lol I not sure lol
